app/uk/gov/hmrc/plasticpackagingtax/returns/config/AppConfig.scala

```diff
@@ -60,6 +60,9 @@ class AppConfig @Inject() (config: Configuration, val servicesConfig: ServicesCo
 
   def pptReturnUrl(id: String): String = s"$pptReturnsUrl/$id"
 
+  def pptGetRegistrationDetailsUrl(pptReference: String): String =
+    s"$pptServiceHost/registrations/$pptReference"
+
```

Comment:
:thinking: I think on the returns service we should always reference it as `subscriptions` and not `registrations`
The call to EIS/ETMP is a subscriptions display which is the domain language we use and I believe that domain language should continue on our code, as even I got a bit confused here :)

Reply:
done